doc/python/parallel-coordinates-plot.md

```diff
@@ -171,6 +171,39 @@ fig = go.Figure(data=
 fig.show()
 ```
 
+### Unselected Line Color and Opacity
+
+
+*New in 5.10*
+
+The color and opacity of unselected lines can be set with `unselected`. Here, we set the color to `lightgray` and the opacity to `0.5`.
```

Comment:
Instead of `lightgray`, I suggest we use another color e.g. `green`.
Also let's add a line to mention that by setting opacity to zero users could hide the deselected lines.

Reply:
Thanks @archmoj , yeah that makes sense. Looks clearer what the behaviour is.